syntax = "proto3";
package util;


import "google/protobuf/timestamp.proto";


message Date {
    int32 year = 1;
    int32 month = 2;
    int32 day = 3;
}


// Oss直传Token
message OssToken {
    // oss access key id
    string access_key_id = 1;
    // 上传参数
    string policy = 2;
    // 签名
    string signature = 3;
    // 过期时间
    google.protobuf.Timestamp expire_time = 4;
    // 上传路径
    string key = 5;
    // 回调地址
    string callback_host = 6;
    // 回调参数
    string callback_body = 7;
}
